tests/e2e/e2e-spec.js

```diff
@@ -0,0 +1 @@
+
```

Comment:
Server initialization part can be included here to get the server up and running before the test execution.

Reply:
This would be needed even for geting the test coverage